solidity/dashboard/src/components/coverage-pools/LearnMoreBanner.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import Banner from "../Banner"
+import * as Icons from "../Icons"
+import NavLink from "../NavLink"
+
+const LearnMoreBanner = ({ onClose }) => {
+  return (
+    <Banner className="banner banner--coverage-pools">
+      <Banner.CloseIcon onClick={onClose} />
+      <div className="flex row space-between full-center">
+        <Banner.Icon icon={Icons.CoveragePool} />
+        <Banner.Title className="h3 text-white">
+          Deposit KEEP in the coverage pool to
+          <br />
+          secure the network and earn rewards.
+        </Banner.Title>
+        <NavLink
+          to="coverage-pools/how-it-works"
+          className="btn btn-tertiary btn-lg"
+        >
+          learn more
+        </NavLink>
```

Comment:
Comparing it to the design in figma i think the banner looks slighty different - mainly because the Banner title is centered while on figma it looks like the title is closer to the icon (so both - icon and title - are kind of closer to the left side) with the `Learn More` button more on the right side.

So I would add margin-left auto to the NavLink to really make it stick to the right side and then add some proper margins to the `Banner.Title` to make it not exactly on center, but almost on center. You see what i'm sayin'? 😅 

When doing that we can also remove the `space-between` class from the parent.